cuqi/geometry/_geometry.py

```diff
@@ -672,15 +685,49 @@ def normalizer(self):
 
     @property
     def coefs(self):
+        # Return None if self.num_modes is 0,
```

Comment:
Before going to explain the special case of num_modes 0, can you please add a general description of what this does.

Reply:
Good point, I added explanation.